grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -78,27 +76,29 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
 
     const cursor = cursorQuery || undefined;
     const start = !isNaN(startQuery) ? Number(startQuery) : 1;
-    const itemsPerPage = !isNaN(perpageQuery) ? Number(perpageQuery) : ITEMS_PER_PAGE;
-
+    const end = !isNaN(perpageQuery) ? Number(perpageQuery) : undefined;
```

Comment:
This will set it to either the query's values or to 1 and undefined, where the end will be determined when fetching the results